packages/test/src/integration-tests.ts

```diff
@@ -1138,6 +1138,32 @@ export function runIntegrationTests(codec?: PayloadCodec): void {
     }
   });
 
+  test('WorkflowClient.signalWithStart fails with WorkflowExecutionAlreadyStartedError', async (t) => {
+    const { client } = t.context;
+    const workflowId = uuid4();
+    const handle = await client.signalWithStart(workflows.sleeper, {
+      taskQueue: 'test',
+      workflowId,
+      signal: 'unblock',
```

Comment:
Not sure if this is the correct signal we should use. I can't run tests locally because I'm unable to do `npm i`.

```
npm ERR! Compiling bridge { target: 'aarch64-apple-darwin', buildRelease: false }
npm ERR! Running /Users/andre.cruz/Work/satazor/sdk-typescript/node_modules/.bin/cargo-cp-artifact [
npm ERR!   '--artifact',
npm ERR!   'cdylib',
npm ERR!   'temporal_sdk_typescript_bridge',
npm ERR!   'releases/aarch64-apple-darwin/index.node',
npm ERR!   '--',
npm ERR!   'cargo',
npm ERR!   'build',
npm ERR!   '--message-format=json-render-diagnostics',
npm ERR!   '--target',
npm ERR!   'aarch64-apple-darwin'
npm ERR! ]
npm ERR! No prebuilt module found at /Users/andre.cruz/Work/satazor/sdk-typescript/packages/core-bridge/releases/aarch64-apple-darwin/index.node
npm ERR! Error: could not find the `cargo` executable.
npm ERR!
npm ERR! You can find instructions for installing Rust and Cargo at:
npm ERR!
npm ERR!     https://www.rust-lang.org/tools/install
npm ERR!
npm ERR!
npm ERR! Did not copy "cdylib:temporal_sdk_typescript_bridge"
npm ERR! /Users/andre.cruz/Work/satazor/sdk-typescript/packages/core-bridge/scripts/build.js:89
npm ERR!     throw new Error(`Failed to build${target ? ' for ' + target : ''}`);
npm ERR!     ^
npm ERR!
npm ERR! Error: Failed to build for aarch64-apple-darwin
npm ERR!     at compile (/Users/andre.cruz/Work/satazor/sdk-typescript/packages/core-bridge/scripts/build.js:89:11)
npm ERR!     at Object.<anonymous> (/Users/andre.cruz/Work/satazor/sdk-typescript/packages/core-bridge/scripts/build.js:106:9)
npm ERR!     at Module._compile (node:internal/modules/cjs/loader:1226:14)
npm ERR!     at Module._extensions..js (node:internal/modules/cjs/loader:1280:10)
npm ERR!     at Module.load (node:internal/modules/cjs/loader:1089:32)
npm ERR!     at Module._load (node:internal/modules/cjs/loader:930:12)
npm ERR!     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
npm ERR!     at node:internal/main/run_main_module:23:47
npm ERR!
npm ERR! Node.js v18.14.0
```

I'm using a apple m1

Reply:
Compilation on m1 is not a problem. You're getting this message because the [rust toolchain](https://rustup.rs/) installed on your computer. You will also need to install the [protobuf compiler](https://github.com/protocolbuffers/protobuf/releases/). For more details on required tools, see the [CONTRIBUTING.md](https://github.com/temporalio/sdk-typescript/blob/main/CONTRIBUTING.md#environment-setup) document, at the root of the project.

Now, don't worry if you're not able to go past this point. I can complete whatever's left.

Reply:
I have followed the guide but I'm still having difficulties. Getting this:

```
 Error: Custom { kind: Other, error: "protoc failed: google/protobuf/duration.proto: File not found.\ngoogle/protobuf/empty.proto: File not found.\ngoogle/protobuf/timestamp.proto: File not found.\ngoogle/protobuf/descriptor.proto: File not found
```

..event though I've put the files under /usr/local/include/ related to the protocol buffers installation.

Reply:
On my local machine, I `protoc` is installed using `brew`.